Library/Formula/mplayer.rb

```diff
@@ -50,19 +39,14 @@ def install
     # It turns out that ENV.O1 fixes link errors with llvm.
     ENV.O1 if ENV.compiler == :llvm
 
-    # we disable cdparanoia because homebrew's version is hacked to work on OS X
-    # and mplayer doesn't expect the hacks we apply. So it chokes.
```

Comment:
I guess it is. Everything that I expected to work works with mplayer and I have literally no idea what "so it chokes" means or for that matter how I could test that